includes/admin/class-wc-admin-post-types.php

```diff
@@ -450,28 +450,29 @@ public function render_product_columns( $column ) {
 	 * @param string $column
 	 */
 	public function render_shop_coupon_columns( $column ) {
-		global $post;
+		global $post, $the_coupon;
```

Comment:
@justinshreve this method is applied for each column, so if is global, we can do only one query for all columns.